Mathlib/RingTheory/PolynomialLaw/Basic.lean

```diff
@@ -32,7 +32,7 @@ polynomials (when the module `M` is free and finite).
 
 ## Implementation notes
 
-In the literature, the theory is writen for commutative rings, but this implementation
+In the literature, the theory is written for commutative rings, but this implementation
 only assume `R` is a commutative semiring.
```

Comment:
Pre-existing: can you change "assume" to "assumes" (as that's the correct grammar)?